docs/sources/mimir/reference-configuration-parameters/index.md

```diff
@@ -737,6 +737,11 @@ forwarding:
   # The CLI flags prefix for this block configuration is:
   # distributor.forwarding.grpc-client
   [grpc_client: <grpc_client>]
+
+# (experimental) Enable marking series as ephemeral based on the given matchers
+# in the runtime config.
+# CLI flag: -distributor.ephemeral-series-enabled
+[ephemeral_metrics_enabled: <boolean> | default = false]
```

Comment:
The YAML config option is still "metrics". Please change it to "series" too.

Reply:
sorry for missing those... fixed: 705f50fcb663fa3cd31021a0187f3dda532501c1